editoast/src/client/mod.rs

```diff
@@ -141,8 +141,8 @@ pub struct RunserverArgs {
     #[derivative(Default(value = r#""amqp://osrd-rabbitmq:5672/%2f".into()"#))]
     #[clap(long, env = "OSRD_MQ_URL", default_value_t = String::from("amqp://osrd-rabbitmq:5672/%2f"))]
     pub mq_url: String,
-    #[derivative(Default(value = "60"))] // TODO: find the currently used timeout
-    #[clap(long, env = "EDITOAST_CORE_TIMEOUT", default_value_t = 60)]
+    #[derivative(Default(value = "180"))] // TODO: find the currently used timeout
```

Comment:
Is that comment still relevant?

Reply:
I don't know, it was added by @Khoyo 
Since he didn't say anything here, maybe it's good ?

Reply:
We did find out and adjusted the prod value, you can remove it.